gitops/snapshot.go

```diff
@@ -1044,6 +1075,22 @@ func GetPRGroupFromSnapshot(snapshot *applicationapiv1alpha1.Snapshot) (string,
 	return "", ""
 }
 
+// GetPipelineAsCodeRepoURLAnnotation gets the value of annotaiton "pac.test.appstudio.openshift.io/repo-url"
```

Comment:
I think you may combine these two functions by passing the annotation name as a parameter

Reply:
That was also my idea, good to see you have the same opinion on that. Will change it